aqua-csp-on-ubuntu-vm/azuredeploy.json

```diff
@@ -0,0 +1,309 @@
+				"description": "The Ubuntu version for the VM. This will pick a fully patched image of this given Ubuntu version. Allowed values: 12.04.5-LTS, 14.04.2-LTS, 15.10."
+			}
+		},
+		"deployAqua": {
+			"defaultValue": "no",
+			"allowedValues": ["no",
+			"yes"],
+			"type": "String",
+			"metadata": {
+				"description": "Specify if you want to deploy Aqua after installing docker"
+			}
+		},
+		"dockerHubUser": {
+			"type": "String",
+			"metadata": {
+				"description": "User name for Docker Hub registry"
+			}
+		},
+		"dockerHubPassword": {
+			"type": "SecureString",
+			"metadata": {
+				"description": "Password for Docker Hub registry"
+			}
+		},
+		"aquaLicenseToken": {
+			"type": "SecureString",
+			"metadata": {
+				"description": "Enter Aqua licnse token"
+			}
+		},
+		"aquaCSPImage": {
+			"defaultValue": "aquasec/csp:3.0.0",
+			"type": "String",
+			"metadata": {
+				"description": "The Aqua CSP version to deploy"
+			}
+		},
+		"aquaAdminPassword": {
+			"type": "SecureString",
+			"metadata": {
+				"description": "Please select your Aqua administrator password"
+			}
+		},
+		"aquaDBPassword": {
+			"type": "SecureString",
+			"metadata": {
+				"description": "Please select the password for PostgreSQL DB that is used during Aqua deployment"
+			}
+		},
+		"addACRReg": {
+			"defaultValue": "no",
+			"allowedValues": ["no",
+			"yes"],
+			"type": "String",
+			"metadata": {
+				"description": "When set to yes, all Azure container registries in the AD will be added to Aqua server"
+			}
+		},
+		"azureADId": {
+			"type": "SecureString",
+			"metadata": {
+				"description": "Azure active directory ID, needs to have contributer role"
+			}
+		},
+		"azureADPassword": {
+			"type": "SecureString",
+			"defaultValue": "",
+			"metadata": {
+				"description": "Azure active directory password"
+			}
+		},
+		"azureTenantID": {
+			"type": "SecureString",
+			"defaultValue": "",
+			"metadata": {
+				"description": "Azure tenant ID"
+			}
+		}
+	},
+	"variables": {
+		"dockerRegistryServer": "docker.io",
+		"extensionName": "CustomScript",
+		"aquaContainerName": "aqua-csp",
+		"imagePublisher": "Canonical",
+		"imageOffer": "UbuntuServer",
+		"sshKeyPath": "[concat('/home/',parameters('adminUsername'),'/.ssh/authorized_keys')]",
+		"OSDiskName": "osdiskforlinuxsimple",
+		"nicName": "myVMNic",
+		"addressPrefix": "10.0.0.0/16",
+		"subnetName": "Subnet",
+		"subnetPrefix": "10.0.0.0/24",
+		"storageAccountType": "Standard_LRS",
+		"publicIPAddressName": "myPublicIP",
+		"publicIPAddressType": "Dynamic",
+		"vmStorageAccountContainerName": "vhds",
+		"virtualNetworkName": "MyVNET",
+		"vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+		"subnetRef": "[concat(variables('vnetID'),'/subnets/',variables('subnetName'))]",
+		"addACRegistries": "https://raw.githubusercontent.com/amitlib/azure-quickstart-templates/aqua-templates/aqua-csp-on-ubuntu-vm/add-acr-registries.json"
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc) for how to handle artifacts